hts-ft/script-hts-ft.js

```diff
@@ -56,23 +54,19 @@ async function scriptHtsFungibleToken() {
         .setInitialSupply(1_000_000)
         // Configure token access permissions: treasury account, admin, freezing
         .setTreasuryAccountId(operatorId)
-        // Set the admin key of the the token to the operator account
-        .setAdminKey(operatorKey) //ToDo: replace with adminKey
         // Set the freeze default value to false
         .setFreezeDefault(false)
-        // Freeze the transaction to prepare for signing
+        //Freeze the transaction and prepare for signing
```

Comment:
for a subsequent PR: something to check and make consistent across the different scripts - let's have standardised comments for the transaction config and flow.
IIRC we made some effort to do this in an earlier round, so they should be mostly consistent now - but probably have drifted apart a little too.